src/App.js

```diff
@@ -1,24 +1,25 @@
 /*eslint-disable*/
-import React from 'react';
-import { Routes, Route, BrowserRouter } from 'react-router-dom';
-// import { Provider } from 'react-redux';
+import React, { useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getBook } from './redux/books/books';
 import Booklist from './components/Booklist';
 import Navbar from './components/Navbar';
 import Categories from './components/Categories';
 
-class App extends React.PureComponent {
-  render() {
+function App(){
+  const dispatch = useDispatch();
+    useEffect(() => {
+      dispatch(getBook());// eslint-disable-next-line
```

Comment:
- [ ] kindly make sure you fix your linters error instead of disabling them as required for every Microverse project and for best practices and clean code. I can see that all your linters are fixable you can just run this command to do so: `npx eslint . --fix`